### PR TITLE
Add controller for /filings and service for Filing Data

### PR DESCRIPTION
**Pull request by PhilipGlover**
- Filing Data controller and service to return Filing data sent to CHIPS filing consumer
-  IDVA3-532
- Docker config required: https://github.com/companieshouse/docker-chs-development/pull/972

Response for Individual:
```
[
    {
        "data": {
            "company_number": "00006400",
            "psc_appointment_id": "01234567",
            "verification_details": {
                "name_mismatch_reason": "PREFERRED_NAME",
                "verification_statements": [
                    "INDIVIDUAL_VERIFIED"
                ],
                "uvid": "999999999"
            }
        },
        "description": "PSC Verfication",
        "description_identifier": null,
        "description_values": null,
        "kind": "psc-verification#psc-verification-individual"
    }
]
```
Response for RLE RO:
```
[
    {
        "data": {
            "company_number": "00006400",
            "psc_appointment_id": "01234567",
            "relevant_officer": {
                "name_elements": {
                    "forename": "Phil",
                    "other_forenames": "Andy",
                    "surname": "Glover",
                    "title": "Mr"
                },
                "date_of_birth": "2000-04-10",
                "is_employee": true,
                "is_director": true
            },
            "verification_details": {
                "verification_statements": [
                    "RO_IDENTIFIED",
                    "RO_VERIFIED",
                    "RO_DECLARATION"
                ],
                "uvid": "999999999"
            }
        },
        "description": "PSC Verfication",
        "description_identifier": null,
        "description_values": null,
        "kind": "psc-verification#psc-verification-rle-ro"
    }
]
```

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -26,3 +26,6 @@ out-of-service.period.message=${PLANNED_MAINTENANCE_MESSAGE:UNAVAILABLE - PLANNE
 
 # VALIDATION STATUS
 feature.flag.transactions.closable=${FEATURE_FLAG_TRANSACTIONS_CLOSABLE_250124:FALSE}
+
+# Filing Data description
+filing.data.description=${PSC_VERIFICATION_DESCRIPTION:PSC Verfication}
```
Comment (AlisonGriffiths) [inline src/main/resources/application.properties:31]: tiny typo - Verification

**Review by AlisonGriffiths — COMMENTED** [review]
Missing a little test coverage for FilingDataConfig and the FilingKind

**Comment by PhilipGlover** [comment]
added tests for `FilingDataConfig` and `FilingKind`

**Comment by ch-code-analysis** [comment]
CI: Security warnings found!

**Review by AlisonGriffiths — APPROVED** [review]

**Pull request merged**